lib/presentation/screens/personal_settings_screen/porsonal_settings_screen.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/presentation/screens/personal_settings_screen/personal_settings_layout.dart';
+
+class PersonalSettingScreen extends StatelessWidget {
+  const PersonalSettingScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const SafeArea(child: PersonalSettingsLayout());
```

Comment:
how many times should I say to reuse safe area? If it is impossible, proof it to me